newrelic.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ["Facelift"],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: "51b50512af4dd4c26e70c0d3a1d84c4b371a812a",
```

Comment:
This feels safer as an env variable.

Reply:
That is, of course, the proper way to do it. :+1: